src/unitxt/processors.py

```diff
@@ -215,3 +215,11 @@ def process_value(self, text: Any) -> Any:
         if self.string.lower() in text.lower():
             return self.string.lower()
         return text
+
+
+class ExtractMtBenchJudgment(FieldOperator):
+    def process_value(self, text: Any) -> Any:
+        match = re.search(r"\[\[([\d]+\.?[\d]*)\]\]", text)
+        if match:
+            return float(match.group(1)) / 10
+        raise RuntimeError(f"Failed to extract judgment ([[number]]) from text {text}")
```

Comment:
will that stops the entire evaluation of the bulk or just for a single instance? It is expected that, for example, 1000 instances, some will fail. So we need to be robust.

Reply:
If the judge fails and we are unable to recover, the evaluation should fail. Otherwise, we will get unpredicted results between runs.

Reply:
Well if the seed is fixed it will be consistnet

Reply:
I just don't think it is reasonable that it will always work for all the instances

Reply:
done